roles/container-engine/containerd/templates/config.toml.j2

```diff
@@ -35,11 +35,9 @@ oom_score = {{ containerd_oom_score }}
 
           [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.{{ runtime.name }}.options]
 {% for key, value in runtime.options.items() %}
-{% if value | string != "true" and value | string != "false" %}
-            {{ key }} = "{{ value }}"
-{% else %}
+{% set is_bool = value|string|lower in ['true', 'false'] %}
```

Comment:
Hey I would prefer the syntax above in the if case, it's more readable IMO, something like that:
```suggestion
{% if value | string | lower in ['true', 'false'] %}
```

Reply:
Please see the next line where `is_bool` gets used.

If I go with your preferred way, the next line will have to look as follows:

```
{% set value = value|lower if (value | string | lower in ['true', 'false']) else '"' ~ value ~ '"' %}
```

which makes it way harder to read.

Reply:
I would only update this line and keep the if/else
```
{% if value | string | lower in ['true', 'false'] %}
            {{ key }} = "{{ value }}"
{% else %}
            {{ key }} = {{ value }}
{% endif %}
```